File: Backend/src/main/java/com/feedle/feedleapi/Networking/SubscribeToUserRequest.java
package com.feedle.feedleapi.Networking;

import com.feedle.feedleapi.Models.UserSubscription;

public class SubscribeToUserRequest extends Request {
    private UserSubscription userSubscription;

    public SubscribeToUserRequest(UserSubscription userSubscription) {
        super(RequestType.SubscribeToUser);
        this.userSubscription = userSubscription;
    }

    public UserSubscription getUserSubscription() {
        return userSubscription;
    }
}

File: FeedleDataTier/FeedleDataTier/Network/SubscribeToUserRequest.cs
using System.Text.Json.Serialization;
using Feedle.Models;

namespace FeedleDataTier.Network
{
    public class SubscribeToUserRequest : Request
    {
        [JsonPropertyName("userSubscription")]
        public UserSubscription UserSubscription { get; set; }
        public SubscribeToUserRequest(UserSubscription userSubscription) : base(RequestType.SubscribeToUser)
        {
            this.UserSubscription = userSubscription;
        }
    }
}